src/opencensus/proto/resource/v1/BUILD.bazel

```diff
@@ -33,9 +33,6 @@ java_proto_library(
 
 go_proto_library(
     name = "resource_proto_go",
-    protos = ["resource.proto"],
-    pb_options = [
-       # omit the go_package declared in proto files to make bazel works as expect
```

Comment:
why did you remove this?

Reply:
I think I need to create a separate Go proto build rule with the paths=source_relative flag and then pass it to the compiler option of go_proto_library.

I'll plan to work on that some time soon.

Reply:
It seems like relative import paths are not yet supported by the `go_proto_library` from io_bazel_rules_go. See https://github.com/bazelbuild/rules_go/issues/1645

Why did we choose relative import paths in the first place?